bigchaindb/config_utils.py

```diff
@@ -100,3 +102,21 @@ def autoconfigure():
     except FileNotFoundError:
         logger.warning('Cannot find your config file. Run `bigchaindb configure` to create one')
 
+
+def get_plugins(plugin_names):
```

Comment:
A couple of improvements:
 - add docstring to describe the function.
 - from line 107 it seems that `plugin_names` can be `None` or an empty list, but when calling the function you must explicitly pass a value. What do you think about changing the signature of the function to `def get_plugins(plugin_names=None)`?